docs/aurora/data-management/moving_data_to_aurora/globus.md

```diff
@@ -1,13 +1,25 @@
-### Transfering files through Globus
+# Transfering files through Globus
 
-Currently only Globus personal is supported 
+Currently, only Globus personal is supported on Aurora. 
+Perform the following steps to transfer data to/from the Aurora login nodes.
 
+1. On a fresh connection to the login nodes, ensuring no proxies are being set (which may require commenting the proxy settings in the `~/.bashrc` or `~/.bash_profile` files), execute
 ```bash
 /soft/tools/proxychains/bin/proxychains4 -f /soft/tools/proxychains/etc/proxychains.conf /soft/tools/globusconnect/globusconnect -setup --no-gui
 ```
-and follow the instruction to setup personal endpoint. 
 
+2. Paste the link provided by the above command in a browser and follow the instructions to setup a personal endpoint
+	* When requested, input your ALCF username and one-time password from your CRYPTOCard/MobilePASS+ token
```

Comment:
spacing issue?

Reply:
This was to create a sub-bullet up item 2. It visualized well for me in the browser when I tested it.